### PR TITLE
Fix selenium tests running locally

### LINKED ISSUE #4536: Fix selenium tests running locally
When working on #4533 I fixed the selenium tests so that they continue to run on travis, but I haven't fixed it running locally. We rarely run selenium tests locally so in the interest of getting that PR ready for review I'm putting the work of fixing selenium in the dev environment in this issue.

### PR DESCRIPTION
**Pull request by umarmughal824**
#### Pre-Flight checklist

- [x] Testing
  - [x] Code is tested
  - [x] Changes have been manually tested

#### What are the relevant tickets?
fixes #4536 

#### What's this PR do?
Fix selenium tests running locally

#### How should this be manually tested?
Just run selenium tests locally

#### Screenshots (if appropriate)
Screenshot 2020-12-02 at 14 14 40
Screenshot 2020-12-02 at 14 18 22
Screenshot 2020-12-02 at 14 14 49

### CODE REVIEW

**Comment by noisecapella** [comment]
Can we make a more specific fix for this issue? I saw only one error:
Screenshot from 2020-12-03 12-45-25

There is a list of ignored console messages here. Can you add a part of the message to this list? https://github.com/mitodl/micromasters/blob/497b331caaeda2e323b6ef20b8227ef3c5dc6036/selenium_tests/util.py#L30

**Comment by umarmughal824** [comment]
@noisecapella Could you please mention the steps to see that warning so that I could test it after adding it in the ignored log messages?

**Comment by noisecapella** [comment]
It should be enough to just run `./script/tests/run_selenium_tests_dev.sh` like you have been. You may need to run migrations first if you have a stale instance: `docker-compose -f docker-compose.yml -f docker-compose.override.yml -f docker-compose.selenium.yml run web ./manage.py migrate`

**Comment by umarmughal824** [comment]
@noisecapella Every time I run that command `/scripts/test/run_selenium_tests_dev.sh`, it never showed that error and every time got passed.

**Comment by umarmughal824** [comment]
@noisecapella waiting for your final review.

File: selenium_tests/basic_test.py
Comment (noisecapella) [inline selenium_tests/basic_test.py:129]: Can we leave out `ignore_errors`?

**Review by noisecapella — CHANGES_REQUESTED** [review]

File: selenium_tests/basic_test.py
Comment (umarmughal824) [reply selenium_tests/basic_test.py:129]: removing it will fail the test.

**Review by noisecapella — APPROVED** [review]

**Pull request merged**
